Watchman.Engine/Generation/Generic/CloudformationStackDeployer.cs

```diff
@@ -264,12 +264,17 @@ private async Task<bool> WaitForStackToReachStatus(string stackName, StackStatus
             } while (true);
         }
 
-        public async Task DeployStack(string name, string body, bool isDryRun)
+        public async Task DeployStack(string name, string body, bool isDryRun, bool onlyUpdateExisting)
```

Comment:
This is a bit of a cryptically named variable. Took me a while to understand what it was actually doing. Maybe `updateOnly` or `noCreate`?